WcaOnRails/app/models/round.rb

```diff
@@ -4,6 +4,10 @@ class Round < ApplicationRecord
   has_one :event, through: :competition_event
   belongs_to :format
 
+  serialize :time_limit, TimeLimit
+  serialize :combined_round_cutoff, CombinedRoundCutoff
+  serialize :next_round_cutoff, Cutoff
```

Comment:
Hm, hm, hm ....

According to the regulations, "_competitors advance to the second phase_" ([9g](https://www.worldcubeassociation.org/regulations/#9g2)) and "_proceed to the next phase_" ([9g2](https://www.worldcubeassociation.org/regulations/#9g2)), so `advancing_cutoff` and `proceed_cutoff` for next round cutoffs might be misleading.

`round_cutoff` and `phase_cutoff` would be simple, but that could also be misinterpreted ("What's the cutoff of the round?" = `phase_cutoff`).

As I said in the issue you linked I support Alberto's suggestion to simply remove the cutoff from 9p2. So maybe `advance_ranking_or_result`?